Ginger/GingerCore/ALM/ALMCore.cs

```diff
@@ -25,13 +26,69 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel;
+using System.Linq;
 
 namespace GingerCore.ALM
 {
 
     public abstract class ALMCore
     {
-        public static ALMConfig AlmConfig = new ALMConfig();
+        //public static ALMConfig AlmConfig = new ALMConfig();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)